### PR TITLE
Code sample - 23c SpringBoot JPA Hibernate UCP

### PR DESCRIPTION
**Pull request by juarezjuniorgithub**
Code sample - 23c SpringBoot JPA Hibernate UCP
Signed-off-by: Juarez Barbosa juarez.barbosa@oracle.com

### CODE REVIEW

**Review by Kuassim — APPROVED** [review]

File: java/23c-springboot3-jpa-ucp/mvnw
Comment (jeandelavarene) [inline java/23c-springboot3-jpa-ucp/mvnw:1]: Is the maven wrapper (third-party) tool required to run these samples? If not, it should be taken out.

**Review by jeandelavarene — CHANGES_REQUESTED** [review]

File: java/23c-springboot3-jpa-ucp/mvnw
Comment (juarezjuniorgithub) [reply java/23c-springboot3-jpa-ucp/mvnw:1]: no, not at all, removed.

**Comment by juarezjuniorgithub** [comment]
@Kuassim @jeandelavarene adjustments executed - indentation fixed. Please review and approve. Thanks.

File: java/23ai-springboot3-jpa-ucp/src/main/java/com/oracle/dev/jdbc/springboot3/jpa/ucp/EmployeeNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.oracle.dev.jdbc.springboot3.jpa.ucp;
+
+class EmployeeNotFoundException extends RuntimeException {
+
+  private static final long serialVersionUID = 7768215094663842819L;
```
Comment (jeandelavarene) [inline java/23ai-springboot3-jpa-ucp/src/main/java/com/oracle/dev/jdbc/springboot3/jpa/ucp/EmployeeNotFoundException.java:26]: Why the serialVersionUID?

File: java/23ai-springboot3-jpa-ucp/pom.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>ojdbc11-production</artifactId>
+      <version>23.4.0.24.05</version>
```
Comment (jeandelavarene) [inline java/23ai-springboot3-jpa-ucp/pom.xml:47]: Please upgrade to 23.6.

Comment (juarezjuniorgithub) [reply java/23ai-springboot3-jpa-ucp/pom.xml]: @Kuassim I have followed Jean's suggestion and updated the driver version to 23.6.0.24.10. 
However, let's try to abide by our latest agreements, as asking recurrently to update the driver version before the very first release of a given code sample configures a dead-end situation, where it's being kept in a situation of open modification. 
Such updates (versions/releases) will be done as part of our agreed Quarterly QA Freshness Pass activity. 
@Kuassim please approve.

File: java/23ai-springboot3-jpa-ucp/src/main/java/com/oracle/dev/jdbc/springboot3/jpa/ucp/EmployeeNotFoundException.java (hunk 1 above)
Comment (juarezjuniorgithub) [reply java/23ai-springboot3-jpa-ucp/src/main/java/com/oracle/dev/jdbc/springboot3/jpa/ucp/EmployeeNotFoundException.java:26]: Removed as requested. @Kuassim please approve, thanks

**Review by jeandelavarene — APPROVED** [review]

**Pull request merged**